<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="2" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="88242" started-at="2016-01-18T19:56:16Z" finished-at="2016-01-18T19:57:44Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="88242" started-at="2016-01-18T19:56:16Z" finished-at="2016-01-18T19:57:44Z">
      <class name="com.dev.GmailTest.GmailTest">
        <test-method status="PASS" signature="SignIn()[pri:0, instance:com.dev.GmailTest.GmailTest@126f7b2]" name="SignIn" duration-ms="43168" started-at="2016-01-19T01:26:16Z" finished-at="2016-01-19T01:26:59Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SignIn -->
        <test-method status="PASS" signature="TearDown(org.testng.ITestResult)[pri:0, instance:com.dev.GmailTest.GmailTest@126f7b2]" name="TearDown" is-config="true" duration-ms="533" started-at="2016-01-19T01:26:59Z" finished-at="2016-01-19T01:27:00Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=SignIn status=SUCCESS method=GmailTest.SignIn()[pri:0, instance:com.dev.GmailTest.GmailTest@126f7b2] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TearDown -->
        <test-method status="FAIL" signature="chekEmail()[pri:0, instance:com.dev.GmailTest.GmailTest@126f7b2]" name="chekEmail" duration-ms="44389" started-at="2016-01-19T01:27:00Z" finished-at="2016-01-19T01:27:44Z">
          <exception class="org.openqa.selenium.UnhandledAlertException">
            <message>
              <![CDATA[Unexpected modal dialog (text: This page is asking you to confirm that you want to leave - data you have entered may not be saved.): This page is asking you to confirm that you want to leave - data you have entered may not be saved.
Build info: version: '2.48.2', revision: '41bccdd10cf2c0560f637404c2d96164b67d9d67', time: '2015-10-09 13:08:06'
System info: host: 'DESKTOP-2B01E58', ip: '192.168.233.1', os.name: 'Windows 10', os.arch: 'x86', os.version: '10.0', java.version: '1.8.0_60'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=43.0.4, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: 4c748fcb-394f-48b6-9bce-042a49f73347
*** Element info: {Using=xpath, value=//div[@class='logo logo-w']}]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.UnhandledAlertException: Unexpected modal dialog (text: This page is asking you to confirm that you want to leave - data you have entered may not be saved.): This page is asking you to confirm that you want to leave - data you have entered may not be saved.
Build info: version: '2.48.2', revision: '41bccdd10cf2c0560f637404c2d96164b67d9d67', time: '2015-10-09 13:08:06'
System info: host: 'DESKTOP-2B01E58', ip: '192.168.233.1', os.name: 'Windows 10', os.arch: 'x86', os.version: '10.0', java.version: '1.8.0_60'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=43.0.4, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: 4c748fcb-394f-48b6-9bce-042a49f73347
*** Element info: {Using=xpath, value=//div[@class='logo logo-w']}
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:206)
	at org.openqa.selenium.remote.ErrorHandler.createUnhandledAlertException(ErrorHandler.java:187)
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:154)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:647)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:353)
	at org.openqa.selenium.remote.RemoteWebDriver.findElementByXPath(RemoteWebDriver.java:490)
	at org.openqa.selenium.By$ByXPath.findElement(By.java:361)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:345)
	at org.openqa.selenium.support.ui.ExpectedConditions.findElement(ExpectedConditions.java:911)
	at org.openqa.selenium.support.ui.ExpectedConditions.access$000(ExpectedConditions.java:41)
	at org.openqa.selenium.support.ui.ExpectedConditions$7.apply(ExpectedConditions.java:205)
	at org.openqa.selenium.support.ui.ExpectedConditions$7.apply(ExpectedConditions.java:201)
	at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:238)
	at com.dev.Library.utils.waitForElementVisible(utils.java:28)
	at com.dev.WebPageFactory.EmailHomePage.signOut(EmailHomePage.java:30)
	at com.dev.GmailTest.GmailTest.chekEmail(GmailTest.java:126)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.UnhandledAlertException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- chekEmail -->
        <test-method status="PASS" signature="TearDown(org.testng.ITestResult)[pri:0, instance:com.dev.GmailTest.GmailTest@126f7b2]" name="TearDown" is-config="true" duration-ms="105" started-at="2016-01-19T01:27:44Z" finished-at="2016-01-19T01:27:44Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=chekEmail status=FAILURE method=GmailTest.chekEmail()[pri:0, instance:com.dev.GmailTest.GmailTest@126f7b2] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TearDown -->
      </class> <!-- com.dev.GmailTest.GmailTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
